core/src/components/datetime/utils/format.ts

```diff
@@ -21,21 +26,25 @@ export const getLocalizedTime = (locale: string, refParts: DatetimeParts, hourCy
     return 'Invalid Time';
   }
 
+  const defaultFormatOptions: Intl.DateTimeFormatOptions = { hour: 'numeric', minute: 'numeric' };
+
+  // If any options are provided, don't use any of the defaults.
+  const options: Intl.DateTimeFormatOptions = formatOptions ?? defaultFormatOptions;
```

Comment:
If `formatOptions` are already typed `Intl.DateTimeFormatOptions` and `defaultFormatOptions` has the same type, we don't need to explicitly assign that type on `const options`.

Reply:
thanks! fixed in b8595a36c5